feature-utils/poly-analysis/src/analysis/report/report-stories.js

```diff
@@ -0,0 +1,21 @@
+export default class ReportStories {
+    constructor(activeReportStories) {
+        this._activeReportStories = activeReportStories;
+    }
+
+    get activeStories() {
+        return this._activeReportStories;
+    }
+
+    get active() {
+        return this.activeStories.length > 0;
+    }
+
+    get jsonReport() {
+        const jsonStoriesReport = this.activeStories.map(
+            (story) => story.jsonReport
+        );
+
+        return { reportAnalyses_v1: jsonStoriesReport };
```

Comment:
Is the format here going to change? I guess until now we were returning for the ID the name of an analysis, and now we return the name of the ministory/report. If yes, we should change to `v2`?